StatsdClient/IOutputChannel.cs

```diff
@@ -1,18 +1,20 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.Threading.Tasks;
 
 namespace StatsdClient
 {
-  /// <summary>
-  /// Contract for sending raw statds lines to the server
-  /// </summary>
-  public interface IOutputChannel
-  {
     /// <summary>
-    /// Sends a line of stats data to the server.
+    /// Contract for sending raw statds lines to the server
     /// </summary>
-    void Send(string line);
-  }
-}
+    public interface IOutputChannel
+    {
+        /// <summary>
+        /// Sends a line of stats data to the server.
+        /// </summary>
+        void Send(string line);
```

Comment:
I don't think this should be in the interface... Send(string) should just be an extension method.. It makes it easier to implementation and then sync method will never change and should call SendAsync().Wait()